public_dropin_gpu_environments/vllm/Dockerfile

```diff
@@ -17,21 +17,24 @@ ENV HOME=/opt/.home
 RUN groupadd -g 1000 datarobot && \
     useradd -u 1000 -g datarobot -m -d $HOME -s /bin/bash datarobot
 
-USER datarobot
+USER 1000:0
 
-COPY --chown=1000:1000 .gitkeep ${HOME}/.placeholder
-COPY --chown=1000:1000 .gitkeep ${CODE_DIR}/.placeholder
-COPY --chown=1000:1000 .gitkeep ${DATAROBOT_VENV_PATH}/.placeholder
+COPY --chown=1000:0 .gitkeep ${HOME}/.placeholder
+COPY --chown=1000:0 .gitkeep ${CODE_DIR}/.placeholder
+COPY --chown=1000:0 .gitkeep ${DATAROBOT_VENV_PATH}/.placeholder
```

Comment:
`datarobot:root` should work as well?

Reply:
Perhaps, but as Erik mentioned (I think I google this as well) that openshift prefers numeric uids/gids instead of names (at least for the USER command). I'm not sure however if this is relevant for COPY --chown however, so decided to stay on the safe side..
https://github.com/datarobot/datarobot-user-models/pull/1371/files

Reply:
Ok, makes sense then